examples/Bremsstrahlung/documentation.yml

```diff
@@ -0,0 +1,59 @@
+--- # Documentation for the Bremsstrahlung Example
```

Comment:
that is wonderful, can you please make it a `README.rst` instead as in #1851?
Sphinx incoming!!!11elf :-)